main.js

```diff
@@ -25,5 +25,6 @@ async function main() {
 try {
     main();
 } catch (error) {
+    core.saveState("EG_FAILED", "true");
```

Comment:
Does saveState only take a string?  I'm curious whether you can pass `true` as opposed to `"true"`.

Reply:
Looking at https://github.com/actions/toolkit/blob/master/packages/core/src/core.ts#L208, I see it being passed around as strings.

Reply:
Well, good news is JS lets you use them interchangeably most of the time!